test/runtests.jl

```diff
@@ -1102,192 +1148,240 @@ end
-                        ("'[2010-01-01 14:30-00, 2010-01-01 15:30-00)'::tstzrange", Interval{Closed, Open}(ZonedDateTime(2010, 1, 1, 14, 30, tz"UTC"), ZonedDateTime(2010, 1, 1, 15, 30, tz"UTC"))),
-                        ("'[2004-10-19 10:23:54-02, 2004-10-19 11:23:54-02)'::tstzrange", Interval{Closed, Open}(ZonedDateTime(2004, 10, 19, 12, 23, 54, tz"UTC"), ZonedDateTime(2004, 10, 19, 13, 23, 54, tz"UTC"))),
-                        ("'[2004-10-19 10:23:54-02, Infinity)'::tstzrange", Interval{Closed, Open}(ZonedDateTime(2004, 10, 19, 12, 23, 54, tz"UTC"), ZonedDateTime(typemax(DateTime), tz"UTC"))),
-                        ("'(-Infinity, Infinity)'::tstzrange", Interval{Open, Open}(ZonedDateTime(typemin(DateTime), tz"UTC"), ZonedDateTime(typemax(DateTime), tz"UTC"))),
-                        ("'[2018/01/01, 2018/02/02)'::daterange", Interval{Closed, Open}(Date(2018, 1, 1), Date(2018, 2, 2))),
-                        # Unbounded ranges
-                        ("'[3,)'::int4range", Interval{Int32}(3, nothing)),
-                        ("'[3,]'::int4range", Interval{Int32}(3, nothing)),
-                        ("'(3,)'::int4range", Interval{Int32, Closed, Unbounded}(4, nothing)),  # Postgres normalizes `(3,)` to `[4,)`
-                        ("'(,3)'::int4range", Interval{Int32, Unbounded, Open}(nothing, 3)),
-                        ("'(,3]'::int4range", Interval{Int32, Unbounded, Open}(nothing, 4)),  # Postgres normalizes `(,3]` to `(,4)`
-                        ("'[,]'::int4range", Interval{Int32, Unbounded, Unbounded}(nothing, nothing)),
-                        ("'(,)'::int4range", Interval{Int32, Unbounded, Unbounded}(nothing, nothing)),
-                        ("'[2010-01-01 14:30,)'::tsrange", Interval{Closed, Unbounded}(DateTime(2010, 1, 1, 14, 30), nothing)),
-                        ("'[,2010-01-01 15:30-00)'::tstzrange", Interval{Unbounded, Open}(nothing, ZonedDateTime(2010, 1, 1, 15, 30, tz"UTC"))),
-                        ("'[2018/01/01,]'::daterange", Interval{Closed, Unbounded}(Date(2018, 1, 1), nothing)),
-                    ]
 
-                    @testset for (test_str, data) in test_data
-                        result = execute(conn, "SELECT $test_str;")
-
-                        try
-                            @test LibPQ.num_rows(result) == 1
-                            @test LibPQ.num_columns(result) == 1
-                            @test LibPQ.column_types(result)[1] >: typeof(data)
-
-                            oid = LibPQ.column_oids(result)[1]
-                            func = result.column_funcs[1]
-                            parsed = func(LibPQ.PQValue{oid}(result, 1, 1))
-                            @test isequal(parsed, data)
-                            @test typeof(parsed) == typeof(data)
-                        finally
-                            close(result)
+                binary_not_implemented_oid = [
```

Comment:
Maybe `pgtypes` instead of `oid`? OID is incorrect here as OID is an integer value.